config/locales/en/global.yml

```diff
@@ -0,0 +1,3 @@
+en:
```

Comment:
We decided not to use translations for any components, as they may be moved out of this repo in future into the main publishing components one, where the translations wouldn’t work.

Reply:
This string isn't used inside the component, just passed into it.

Reply:
Since this is an interactive element, you don't need to use a translation for it.